decidim/decidim
# Fix empty address in conference's venues

#### :tophat: What? Why?
This PR hides the venues address when the meeting address is empty. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #11275

#### Testing
1. Login as admin, and visit conference admin 
2. Unpublish meeting component 
3. Create a new Meeting component, and publish 
4. Create 1 online meeting 
5. Create a hybrid meeting
6. Check the frontend ... you should be able to see empty box 
7. Apply patch 
8. Refresh 
9. See empty box is gone
 

:hearts: Thank you!
